app/components/dashboard/dashboard.html

```diff
@@ -75,6 +75,13 @@
               <td>Nodes in the cluster</td>
               <td>{{ infoData.Swarm.Nodes }}</td>
             </tr>
+            <tr>
+              <td colspan="2">
+                <div class="btn-group" role="group" aria-label="...">
+                  <a class="btn btn-outline-secondary" type="button" ui-sref="swarm.visualizer"><i class="fa fa-object-group space-right" aria-hidden="true"></i>Go to cluster visualizer</a>
```

Comment:
`btn-outline-secondary` not available in Bootstrap 3.

Reply:
Good point, will fix as a part of #1460

Reply:
Fixed in https://github.com/portainer/portainer/pull/1460/commits/5fc1626a28ee8faf7b71e2220713118146467e2d